swap_meet/vendor.py

```diff
@@ -1,2 +1,83 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        # Add an item to inventory
+        if item:
+            self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        # Remove an item from inventory
+        try:
+            self.inventory.remove(item)
+            return item
+        except ValueError as err:
+            print(f"{err}. Unable to remove '{item}' from inventory.")
+            return False
+
+    def get_by_id(self, id):
+        # Return item that matches target id
+        try:
+            for item in self.inventory:
+                if item.id == id:
+                    return item
+        except IndexError as err:
```

Comment:
There is nothing in the try block that could throw an IndexError. This area of the code will never run.